src/components/body/Body.tsx

```diff
@@ -4,7 +4,11 @@ import { EmojiList } from './EmojiList';
 import { useEffect, useRef } from 'react';
 import { useActiveCategoryState } from '../contextProvider/PickerContextProvider';
 
-export function Body() {
+interface Props { 
+  emojiListRef: any;
+}
```

Comment:
Rather than using `any`, better use the actual ref type from React:
```suggestion
type Props = { 
  emojiListRef: React.RefObject<HTMLElement>;
}
```